eureka2-core/src/main/java/com/netflix/eureka2/config/EurekaTransportConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.netflix.eureka2.config;
+
+import com.netflix.eureka2.transport.EurekaTransports;
+
+/**
+ * @author David Liu
+ */
+public interface EurekaTransportConfig {
```

Comment:
It is more about client API usability. With builder we can have more fluent
object construction.

On Mon, Feb 9, 2015 at 1:00 PM, qiangdavidliu <notifications@github.com>
wrote:

> In
> eureka2-core/src/main/java/com/netflix/eureka2/config/EurekaTransportConfig.java
> <https://github.com/Netflix/eureka/pull/404#discussion_r24364815>:
>
> > @@ -0,0 +1,13 @@
> > +package com.netflix.eureka2.config;
> > +
> > +import com.netflix.eureka2.transport.EurekaTransports;
> > +
> > +/**
> > + * @author David Liu
> > + */
> > +public interface EurekaTransportConfig {
>
> There are only two variables right now. We can add a builder if this
> config grows in the future.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Netflix/eureka/pull/404/files#r24364815>.
>